### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by robertgarrigos** [community/community#66041] [Actions] [answered]
### Select Topic Area

Bug

### Body

I'm having an error 500 when accessing my profile page or trying to update a repo description. Other pages are ok.

**Comment by robertgarrigos** [comment]
I had to send this message several times. It seems fixed, now.

**Comment by PmnAngelov** [comment, marked as answer]
If you're experiencing a 500 Internal Server Error it's likely an issue on GitHub's end. 

Sometimes, server errors are temporary and can be resolved by waiting for a while and then refreshing the page. For the future, you can also check GitHub's status page (https://www.githubstatus.com/) to see if there are any ongoing incidents or outages reported.

If the issue persists, and there are no reported incidents on GitHub's status page, you can also consider reaching out to the guys at GitHub Support.

**Comment by gfreitasb** [comment]
> ### Selecione a área do tópico
> Erro
> 
> ### Corpo
> Estou tendo um erro 500 ao acessar minha página de perfil ou ao tentar atualizar a descrição de um repositório. Outras páginas estão ok.

Estou na mesma situação ao criar um repostório

**Comment by thinker-kim** [comment]
Same issue is occurring in the class where I'm a TA as well...

**Comment by joshjordan** [comment]
Getting 500s but status page says all systems operational

![image](https://github.com/user-attachments/assets/acbc6058-4751-4c0c-9aa9-5b67847d5e78)

**Comment by eliliam** [comment]
Same issue when adding a reaction to a comment
